providers/ibm-cloud-natural-language-understanding-eu-de.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "ibm-cloud-natural-language-understanding-eu-de",
+  "services": [
+    {
+      "id": "default",
+      "baseUrl": "https://api.eu-de.natural-language-understanding.watson.cloud.ibm.com"
```

Comment:
can we use the temporary workaround with the input variables (see the discussion in slack)?

Reply:
yea that looks good !

Reply:
I removed region from provider name and injected it using input variables.

Reply:
@zdne But I am not sure about the solution. Latest proposal from @smizell disallows using inputs in provider url definitions. See: https://www.notion.so/superface/Immutable-providers-a4d9b3b4af3f4635973315393b631407#f53f7f929d3647d19da01f51e3a38f52